blog/2025-10-15-hacktoberfest

```diff
@@ -0,0 +1,81 @@
+---
+slug: hacktoberfest
```

Comment:
Please correct the file name. It should be:

`blog/2025-10-15-hacktoberfest.mdx`

Reply:
maybe?? but I don't think you have to tbh. The slug looks okay.. you might want to update just because next year you will want to use that slug again so that the URL is clean (blog/hacktoberfest), right? But you can always change this file next year (blog/hacktoberfest25).  It's not necessary... up to you